recipes/pandora/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "pandora" %}
+{% set version = "0.9.1" %}
+{% set repo_URL = "https://github.com/rmcolq/pandora" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: {{ repo_URL }}/releases/download/{{ version }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 1dbde37526f1d2d0b36b51c5366a6ef96a975a45127dd265aec5d61ccf00caa8
+  
+build:
+  number: 0
+  skip: True #  [osx]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake
+    - make
+    - wget
+    - mafft
```

Comment:
Does it need mafft during run-time?

Reply:
Hello,

sorry for the late reply. We are running a [sample example test](https://github.com/rmcolq/pandora/tree/master/example) just after building the executable as an extra step to make sure things are fine. To run this sample example, `pandora` requires `make_prg` (another tool), that has as dependencies `wget` and `mafft`. Even though `make_prg` is conda installable, we are just downloading the binary using `wget` because we use a specific version of `make_prg` that is still unstable. It will take some months until it gets stable, and merged into the main branch. At that point, we will update the `make_prg` recipe, and this one. This one shall thus have `make_prg` and `mafft` on build and runtime steps... but until then we will install `make_prg` without conda and use `mafft` just in the build step.

Does this sound reasonable, or should I update the recipe?

Reply:
ah, gotcha, fair enough